### PR TITLE
Update dock article

### PR DESCRIPTION
**Pull request by ahoneybun**
This updates the content and screenshots including the new steps for Ubuntu 22.04.

### CODE REVIEW

**Review by thomas-zimmerman — DISMISSED** [review]
npm live view looks good.

**Review by thomas-zimmerman — APPROVED** [review]
Nice update to the article

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:62]: >edge to display the launcher on

Probably want to change "launcher" to "dock" here. Launcher is still something else.

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:74]: s/effect/affect/g

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:76]: >the GNOME Shell <u>Extension</u> application

s/Extension/Extensions/g

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:104]: >Then enable Manually Installed extensions 

Should this be "scroll to Manually Installed"? It doesn't look like there's anything to enable in the "Manually Installed" section in that screenshot.

**Review by leviport — CHANGES_REQUESTED** [review]

File: content/customize-dock.md
Comment (ahoneybun) [reply content/customize-dock.md:104]: This looks fine to me in the live version.

Screenshot from 2023-09-28 07-38-41

File: content/customize-dock.md
Comment (ahoneybun) [reply content/customize-dock.md:34]: Wouldn't this also be the same for Ubuntu? Do we just want to get rid of Dash to Dock completely?

File: content/customize-dock.md
Comment (leviport) [reply content/customize-dock.md:104]: I can't figure out what the sentence is telling me to do. If I can't figure it out, someone else is going to struggle to understand it as well. It tells me to enable two different things, and I'm not sure what the first thing is that it wants me to enable.

File: content/customize-dock.md
Comment (leviport) [reply content/customize-dock.md:34]: Dash to Dock is probably less likely to have compatibility issues on Ubuntu, but I guess that would be fine. Your call, I guess.

File: content/customize-dock.md
Comment (ahoneybun) [reply content/customize-dock.md:104]: Ah I think I see the issue, I think the Extensions at the top was disabled which caused Dash to Dock to be greyed out.

File: content/customize-dock.md
Comment (ahoneybun) [reply content/customize-dock.md:34]: I don't see any options for COSMIC Dock in dconf only dash-to-dock stuff so I don't think we expose those options. So perhaps Dash to Dock is better when you need more options.

Comment (leviport) [reply content/customize-dock.md:34]: They're under dash-to-dock in dconf-editor. They're all there.

**Comment by ahoneybun** [comment]
I made changes to remove Dash to Dock installation and use dconf Editor instead per @leviport 's suggestion.

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:26]: >Pop!\_OS and Ubuntu both contain an application dock on the bottom of the screen.

It's on the left side for Ubuntu. I think they should either have their locations mentioned separately so that they are both correct, or the locations should be omitted entirely.

>move the dock to the bottom of the screen,

If the previous sentence says it's already on the bottom edge of the screen, it doesn't make sense to say that it can be moved to the bottom of the screen (it's already there). Something like "change the location of the dock" seems like it would fit better.

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:36]: I think Dash to Dock sets the location to the bottom edge by default, so saying that's why it's on the left edge seems confusing.

**Review by leviport — CHANGES_REQUESTED** [review]

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:27]: >You can adjust this preference, such as move the dock to a different side the screen, and more in <u>Settings</u>.

"such as" makes this sentence not work, and it's unclear what "this preference" refers to, since the position was removed from the previous sentence.

If I were to rewrite it, I'd go with something like this:

>The location of the dock, as well as other dock settings, can be adjusted in Settings.

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:31]: "the screenshot below" wasn't mentioned before. Why was that added? 

Also I don't think either COSMIC Dock or Ubuntu's dock are any lighter than Dash to Dock, so I'm not so sure "lightweight" has any basis in this note or the one below. But I've been overlooking that on my past reviews, so if you want to leave it, I won't complain too much. I'm just uncertain about it.

**Review by leviport — CHANGES_REQUESTED** [review]

**Review by leviport — APPROVED** [review]

File: content/customize-dock.md
Comment (leviport) [inline content/customize-dock.md:31]: I already approved this PR, but I think it might be to change this to: 

>Pop!_OS's dock is positioned along the bottom edge of the screen by default.

Just because "lightweight" still doesn't make sense to me and the sentence feels like a placeholder. Up to you whether or not you want to implement it. Obvs. you'd want to make the Ubuntu one below match this one.

Comment (ahoneybun) [reply content/customize-dock.md]: Should be updated now so we can get another approval on this PR.

**Review by thomas-zimmerman — APPROVED** [review]
LGTM

**Pull request merged**
